great_expectations/experimental/datasources/data_asset/data_connector/filesystem_data_connector.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import annotations
```

Comment:
This entire module was developed from scratch to implement the interfaces in the parent classes.  It is qualitatively different than its V3 counterpart and much simpler, because it only relates to ConfiguredAsset DataConnector model (please see the explanation above).This entire module was developed from scratch following the spirit of the relevant parts of the V3 interface design; however, it is vastly simpler, because it only relates to ConfiguredAsset DataConnector (please see the explanation above). The implementation utilizes a V3 by importing and reusing it (the utility was modified to serve both V3 and ZEP use cases).